priv/repo/migrations/20211014073049_add_web_auth_requests_table.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Hexpm.RepoBase.Migrations.AddWebAuthRequestsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:requests) do
+      add(:device_code, :string)
+      add(:user_code, :string)
+      add(:scope, :string)
+      add(:verified, :boolean)
+      add(:user_id, :integer)
```

Comment:
```suggestion
      add(:user_id, references(:users), null: false)
```

I believe we should have `null: false` on all fields?

Reply:
> I believe we should have null: false on all fields?

Yes.